FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -34,8 +35,421 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        ተ፡ ጠር <gap reason="illegible"/> ንዱ <gap reason="illegible"/> ሀሎ፡ ውስተ፡ ቤተ፡ <gap reason="illegible"/></foreign>, </desc>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#ii"/>
+                                    <desc type="StampExlibris"><foreign xml:lang="la">"Biblioth Palatin Medicae II 55 403"</foreign> written in pencil on top of the recto of the second 
+                                        protection leaf, that is marked with "I".</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">"11 55" and "40" written by an early European hand on labels on the top and at the bottom of the spine.</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#vi"/>
+                                    <desc type="StampExlibris"><foreign xml:lang="it">"Preghiere incantatorie in lingua abissina"</foreign> written in pen at the top of the verso of the sixth protection leaf, 
+                                        that is marked with "V". Written with similar hand as the foliation marks (1-209) and which appears to be the same as the one in <ref target="BMLor148"/> 
+                                        by <ref target="PRS10036Wansleb">Johann Michael Wansleben</ref>.</desc>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#214v"/>
+                                    <desc type="StampExlibris">"376" and "27" written in pen in very tall size and by a similar hand as the foliation marks and <ref target="#e3"/>.</desc>
+                                </item>
+                                <item xml:id="e5">
+                                    <locus target="#ii"/>
+                                    <desc type="StampExlibris">"Pal. Or. 403" written in blue pencil on bottom of the recto of the second protection leaf, that is marked with "I".</desc>
+                                </item>
+                                <item xml:id="e6">
+                                    <locus target="#i"/>
+                                    <desc><foreign xml:lang="it">"I l. La preghiera della Madonna a Bartos 49 preghiere 80 preghiere della Madonna sul Golgota 110 preghiere 
+                                        alla Madonna 146 id. (per la Domenica) 159 id. 164 id. 174 Preghiera di S. Cipriano 197v preghiera contro il demonio, le malattie ecc."</foreign>,
+                                        written on a leaflet attached on the back side of the first, not numbered protective leaf. The backside of the leaflet containing the 
```

Comment:
unnumbered

Reply:
Yes. I need to admit, that it is unnumbered insofar as it carries no number written by pencil in the manuscript, but in the TEI encoded record on Beta maṣaḥəft it carries the number "i". I hope, that I made that clear somehow, but I think it will confuse many people. A similar problem is with the folios beginning 173. Because some folios are double (173, 173bis) the number on the parchment does not match to the number in the record. I encoded this information, but users will struggle with that confusion nevertheless.

Reply:
It is a common problem, I think the best we can do is to be as explicit as possible (there are no ideal solutions).